extension/_locales/en-US/messages.json

```diff
@@ -68,7 +68,7 @@
       }
     },
     "displayStatisticsMessage": {
-      "message": "Display Statistics",
+      "message": "Display Statistics?",
```

Comment:
Given the screenshot, I'm not completely sure it needs the question mark, and needs a fix for the comment instead.

Reply:
What happens when you click that command? I assume it opens a page for the statistics.

My suggestion was about the comment (i.e. description): `Command used to show statistics (display is intended as a verb)`.

About the message itself: if I my assumption is correct, I don't think it needs a question mark.

Reply:
> What happens when you click that command? I assume it opens a page for the statistics.

It shows the statistics in the info-bar itself. Attaching screenshot for details.

<img width="1280" alt="Screenshot 2022-02-22 at 15 03 22" src="https://user-images.githubusercontent.com/66322306/155148103-156b729d-de3b-4887-9c9b-be818fa11c6a.png">

Reply:
Ah, OK. I was expecting a pop-up, or similar. Then maybe update the comment to `Option to show statistics in the infobar (display is intended as a verb).`?

Reply:
Yeah, sounds good to me.